src/v/syschecks/syschecks.cc

```diff
@@ -20,24 +20,56 @@
 #include <seastar/core/seastar.hh>
 #include <seastar/net/api.hh>
 
+namespace {
+ss::sstring to_string(ss::fs_type fs) {
+    return [fs]() {
```

Comment:
What's the advantage of wrapping this in a lambda?

Reply:
No advantage! Originally this was an `operator<<(ostream&, fs_type)` overload and in that case it allowed each case statement to `return` rather than needing an assignment & `break`, but after a change this method just returns the string directly now it's no longer needed. Fixing.

Reply:
The extraneous lambda is removed.